clustering/clustering.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+
+from __future__ import print_function
+import jubatus
+from jubatus.common import Datum
+
+datasets_name = ["noisy_circles", "noisy_moons", "blobs"]
+datasets_directory = "./datasets/"
+
+datasets = {}
+for name in datasets_name:
+    dataset = []
+    with open(datasets_directory+name+".dat", "r") as f:
+        for line in f:
+            cluster, x, y, = line.rstrip().split(",")
+            dataset.append({"x":float(x), "y":float(y), "cluster": int(cluster)})
+    datasets[name] = dataset
+
+print("loaded three datasets(./datasets/{{{0[0]},{0[1]},{0[2]}}}.csv). Each dataset has two cluster. See ./expect/{{{0[0]},{0[1]},{0[2]}}}.png".format(datasets_name))
+
+host = '127.0.0.1'
+port = 9199
+name = 'test'
+
+client = jubatus.Clustering(host, port, name)
+
+print("excute clustering the datasets")
+
+try:
+    import matplotlib.pyplot as plt
+    import numpy as np
+    isVisualization = True
+    colors = np.array([x for x in 'bgrcmykbgrcmykbgrcmykbgrcmyk'])
+    colors = np.hstack([colors] * 20)
+    plt.figure(figsize=(len(datasets_name) * 2 + 6, 4))
+except ImportError:
+    print("no matplotlib or numpy, so can't visualize clustering")
+    isVisualization = False
```

Comment:
CamelCase and snake_case are mixed. 
Should we write `is_visualization` or simply `visualization`?